Translated/Tasks/Side/ss0030766051.csv

```diff
@@ -1,4 +1,4 @@
 Title#0,"""Making Multi-Weapons"""
-Summary#0,"""Staggy, an ARKS member,<br>teaches you how to make Multi-Weapons<br>at the Item Lab.<br><br><yellow>※Recommended Gathering Sections：<br>Nothern Aerio<c><c>"""
+Summary#0,"""Staggy, an ARKS member,<br>teaches you how to make Multi-Weapons<br>at the Item Lab.<br><br><yellow>※Recommended Gathering Sections：<br>Northern Aerio<c><c>"""
```

Comment:
Double <c> tag here. Shouldn't cause problems but not necessary

Reply:
do you refer to the double line break or the <c> tag here?

Reply:
The <c> at the end.